src/js/common/schemaform/FormApp.jsx

```diff
@@ -1,221 +1,55 @@
 import React from 'react';
-import moment from 'moment';
-import Scroll from 'react-scroll';
-import { withRouter } from 'react-router';
 import { connect } from 'react-redux';
 
 import FormNav from './FormNav';
 import FormTitle from './FormTitle';
 import AskVAQuestions from './AskVAQuestions';
-import {
-  LOAD_STATUSES,
-  PREFILL_STATUSES,
-  SAVE_STATUSES,
-  setFetchFormStatus,
-  fetchInProgressForm
-} from './save-load-actions';
-import LoadingIndicator from '../components/LoadingIndicator';
-
-import { isInProgress } from '../utils/helpers';
-
-const Element = Scroll.Element;
-const scroller = Scroll.scroller;
-const scrollToTop = () => {
-  scroller.scrollTo('topScrollElement', window.VetsGov.scroll || {
-    duration: 500,
-    delay: 0,
-    smooth: true
-  });
-};
-
-moment.updateLocale('en', {
-  meridiem: (hour) => {
-    if (hour < 12) {
-      return 'a.m.';
-    }
-    return 'p.m.';
-  },
-  monthsShort: [
-    'Jan.',
-    'Feb.',
-    'Mar.',
-    'Apr.',
-    'May',
-    'June',
-    'July',
-    'Aug.',
-    'Sept.',
-    'Oct.',
-    'Nov.',
-    'Dec.'
-  ]
-});
+import RoutedSavableApp from './RoutedSavableApp';
 
 /*
  * Primary component for a schema generated form app.
  */
 class FormApp extends React.Component {
-  componentWillMount() {
-    window.addEventListener('beforeunload', this.onbeforeunload);
-    if (window.History) {
-      window.History.scrollRestoration = 'manual';
-    }
-
-    // If we start in the middle of a form, redirect to the beginning or load
```

Comment:
Not sure if we want to keep any of this functionality in the case of save being disabled. Right now if it's an anonymous form and they somehow arrive at a url in the middle of the form they won't be redirect.

Reply:
I think we can leave it out for now. We might be able to remove that functionality entirely if we work on the issue to show validation errors on the review page.